opm/material/common/Unused.hpp

```diff
@@ -40,4 +40,10 @@
 #define OPM_OPTIM_UNUSED
 #endif
 
+#ifdef HAVE_MPI
+#define OPM_UNUSED_MPI
+#else
+#define OPM_UNUSED_MPI OPM_UNUSED
```

Comment:
I read the macro name as "unused if we have MPI" but the meaning is "unused if we do NOT have MPI". This is opposite of for example `OPM_DEBUG_UNUSED` which means "unused in debug mode".

I therefore suggest `OPM_UNUSED_NOMPI` for this. (If we ever need to add the counterpart it would be `OPM_UNUSED_MPI`.)

Reply:
i fgured we'll never ever need the opposite macro, and thus it's just a longer thing to type. but okay, will rename in the name of consistency.